lm_diagnoser/example_setup_extract.json

```diff
@@ -0,0 +1,12 @@
+{
+  "model": "../models/gulordava/hidden650_batch128_dropout0.2_lr20.0.cpu.pt",
+  "vocab": "../models/gulordava/vocab.txt",
+  "load_modules": "../models/gulordava/",
```

Comment:
In principle I updated everything, but I relaxed the assumption that required modules (such as model.py) should be in the same folder as the model itself (in case, for instance, I have several models that require the same baseclass, but are located in different folders). I updated it in the ForwardLSTM as well. Does that address your concern, or did I misunderstand what you meant?

Reply:
Ah yes, that's fine. I meant mostly that the argument is now called `load_modules` here, and still referred to as `module_path` in the `ForwardLSTM` init.

Reply:
Ok, I think at least it shouldn't cause any troubles then. We can change the name again if you like it better.